design/baremetal-operator/explicit-boot-mode.md

```diff
@@ -0,0 +1,117 @@
+<!--
+ This work is licensed under a Creative Commons Attribution 3.0
+ Unported License.
+
+ http://creativecommons.org/licenses/by/3.0/legalcode
+-->
+
+# explicit-boot-mode
+
+## Status
+
+implementable
+
+## Summary
+
+This design extends the boot mode handling described in
+[implicit-boot-mode](implicit-boot-mode.md) by adding a field for the
+user to set the boot mode explicitly.
+
+## Motivation
+
+As was pointed out late in the [implicit-boot-mode
+review](https://github.com/metal3-io/metal3-docs/pull/78), we cannot
+always assume that old BMC protocols and old boot modes are
+automatically combined. We still want to provide a reasonable default
+behavior that encourages and assumes UEFI, but does not prevent the
+use of legacy boot mode.
+
+### Goals
+
+- Describe an API change to allow the user to override the default
+  boot mode selected based on the BMC type.
+
+### Non-Goals
+
+- Change the implicit boot mode selection design.
+
+## Proposal
+
+Add a new optional API input field, `spec.bootMode`, with possible
+values `UEFI` or `legacy`. If no value is provided, the value from the
+BMC method will be used.
```

Comment:
Maybe "the system running default will be used."

Which would be the truth, the BMC doesn't entirely govern system configuration, it just provides a window and set of knobs.

Reply:
Here I'm talking about the method on the BMC type, not the actual hardware management controller.